README.md

```diff
@@ -0,0 +1,33 @@
+# hub-system-charts
+This repository is a helm chart that contains all the charts that are used by provisioned Hub clusters.
+
+## Charts
+The `charts` directory contains forked charts that require edits beyond the scope that their values originally provided. If at all possible,
+we should avoid forking charts and should instead the values should be updated to provide the desired functionality in `templates`. However, 
+if that is not possible, the forked chart should be kept as close to the original as possible.
+
+To package all these `charts`, run `./scripts/package.sh`. This will package the charts into the `docs` directory so long as the script
+has been updated to be aware of any new charts.
+
+### Istio
+For the purposes of using istio withing provisioned clusters, we have forked the upstream `Istiod` chart and setup:
+
+1. Two replicas for the `Istiod Deployment` that use `podAntiAffinity` to keep one on a master node and the other on worker nodes.
+3. The Istiod Service in the `istio-system` namespace to only select the `Istiod` pods with the `on-master: "false"` label.
+4. A new Service in the `istio-system` namespace called `istiod-remote` that only selects `Istiod` pods with the `on-master: "true"` label.
+5. The `MutatingWebhookConfiguration` to send requests to the `istiod-remote` `Service` only.
```

Comment:
Markdown will let you start every item in an ordered list with `1.` and when it renders it will set all the numbers correctly.

```suggestion
1. Two replicas for the `Istiod Deployment` that use `podAntiAffinity` to keep one on a master node and the other on worker nodes.
1. The Istiod Service in the `istio-system` namespace to only select the `Istiod` pods with the `on-master: "false"` label.
1. A new Service in the `istio-system` namespace called `istiod-remote` that only selects `Istiod` pods with the `on-master: "true"` label.
1. The `MutatingWebhookConfiguration` to send requests to the `istiod-remote` `Service` only.
```

Reply:
How have I never learned this?!

Reply:
That's very cool.